build/_includes/documentation_menu.njk

```diff
@@ -23,10 +24,24 @@
       <li><a href="{{ DOCS_BASE }}#Uploading-media">Uploading media </a></li>
       <li><a href="{{ DOCS_BASE }}#Using-media-from-third-party-providers">Using media from third-party providers </a></li>
       <li><a href="{{ DOCS_BASE }}#Add-a-background-image-or-video">Add a background image or video</a></li>
+      <li><a href="{{ DOCS_BASE }}#Add-and-manage-page-layouts">Create &amp; manage page layouts</a></li>
       <li><a href="{{ DOCS_BASE }}#Masking-cropping">Masking &amp; cropping</a></li>
-      <li><a href="{{ DOCS_BASE }}#Adding-links">Adding links &amp;</a><a href="#Add-a-page-attachment"> page attachments</a></li>
-      <li><a href="{{ DOCS_BASE }}#Saving-reusing-styles-and-colors">Saving &amp; reusing styles and colors </a></li>
-      <li id=-"t2" class="docTrack" aria-hidden="true"></li>
+      <li><a href="{{ DOCS_BASE }}#Adding-text">Adding text</a></li>
+      <li><a href="{{ DOCS_BASE }}#Adding-links">Adding links</a>
+      <li><a href="{{ DOCS_BASE }}#Animate-page-elements">Animate page elements</a></li>
+      <li><a href="{{ DOCS_BASE }}#Add-borders">Add borders &amp; corner radius</a></li>
+      <li><a href="#Add-a-page-attachment">Add a page attachment</a></li>
+      <li><a href="{{ DOCS_BASE }}#Saving-reusing-styles-and-colors">Saving &amp; reusing styles &amp; colors </a></li>
+      <li><a href="#Save-your-progress">Save your progress</a></li>
```

Comment:
We're switching a lot between e.g. "Adding" and "Add". Mostly a nit, just curious if you think we should clean that up